src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+![A support scam on Discord](./supportScam.png)
+
+As a general rule, staff will never communicate with you through private, unofficial channels. Some simple things to keep in mind when dealing with support:
+
+- Never share your private keys, seed phrases or passwords
+- Never allow anyone remote access into your computer
+- Never communicate in unofficial private channels
+
+<InfoBanner emoji=":lock:">
+  <div>
+    Beware: although support-style scams commonly happen on Discord, they can also be prevalent on any chat applications where crypto discussion happens, including email.
+  </div>
+</InfoBanner>
+
+### Phishing Scams
+
+Phishing scams are another increasingly common angle that scammers will use to attempt to steal your wallet's funds.
+
+Some phishing emails ask users to click on links that will re-direct them to imitation websites, asking them to enter their seed phrase, reset their password or send ETH. Others may ask you to unknowingly install malware to infect your computer and give scammers access to your computer's files.
+
+If you receive an email from an unknown sender, remember:
+
+- Never open a link or attachment from email addresses you don't recognise
+- Never divulge your personal information or passwords to anyone
+- Delete emails from unknown senders
+
+### Crypto trading broker scams
+
+Scam crypto trading brokers claim to be specialist cryptocurrency brokers who will offer to take your money and invest it on your behalf. The promises of unrealistic returns usually accompany this offer. After the scammer receives your funds, they may lead you on, asking that you send more funds, so you don't miss out on further investment gains, or they may disappear entirely.
+
+These fraudulent brokers find their targets by using fake accounts on YouTube to start seemingly natural conversations about the broker. These conversations are often highly upvoted to increase legitimacy, but the upvotes are all from bot accounts.
+
+**Do not trust internet strangers to invest on your behalf. You will lose your crypto.**
+
+![A trading broker scam on YouTube](./brokerScam.png)
+
+<Divider />
+
+## Further Reading
+
+### Web Security
+
+- [This is why you shouldn’t use texts for two-factor authentication](https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin)
+- [Up to 3 million devices infected by malware-laced Chrome and Edge add-ons](https://arstechnica.com/information-technology/2020/12/up-to-3-million-devices-infected-by-malware-laced-chrome-and-edge-add-ons/)
+- [How to Create a Strong Password — That You Won’t Forget](https://www.avg.com/en/signal/how-to-create-a-strong-password-that-you-wont-forget)
+
+### Crypto Security
+
+- [Crypto Security: Passwords and Authentication](https://www.youtube.com/watch?v=m8jlnZuV1i4)
+- [4 Ways to Stay Safe in Crypto](https://www.coindesk.com/tech/2021/04/20/4-ways-to-stay-safe-in-crypto/)
+
```

Comment:
```suggestion
- [MyCrypto’s Security Guide For Dummies And Smart People Too](https://medium.com/mycrypto/mycryptos-security-guide-for-dummies-and-smart-people-too-ab178299c82e)

```
Here's another in-depth crypto security guide we could add.

Reply:
Let's do it